app-admin/systemrescuecd-x86/systemrescuecd-x86-4.6.0.ebuild

```diff
@@ -18,8 +18,14 @@ S=${WORKDIR}
 RESTRICT="mirror"
 
 src_install() {
-	insinto "/usr/share/${PN%-*}"
-	doins "${DISTDIR}/${P}.iso"
+	insinto /usr/share/${PN%-*}
+	doins ${DISTDIR}/${P}.iso
```

Comment:
Thou shalt not remove valid quoting. Even if you believe `${DISTDIR}` can't contain spaces.